emails/views.py

```diff
@@ -784,7 +783,7 @@ def _handle_received(message_json: AWS_SNSMessageJSON) -> HttpResponse:
     if has_text:
         incr_if_enabled("email_with_text_content", 1)
     if issues:
-        info_logger.warning(
+        info_logger.info(
```

Comment:
question (non-blocking): Looks like this line should have been logging at the info level all along?

Reply:
there's some rules about the loggers that don't seem to be documented anywhere. Only logging 'info` might be one of them.